src/components/EntityInfo.vue

```diff
@@ -41,7 +41,7 @@
 
             <span v-if="isHistorical">
               <v-chip class="primary mt-n1 ml-4" small label text-color="white">HISTORICAL</v-chip>
-              <span class="font-14 ml-3">{{dissolutionText}}</span>
+              <span class="font-14 ml-3">{{historicalText}}</span>
```

Comment:
`historicalText` will be constructed from the "state filing" that caused the change. Right now we only use this for Historical state, but this is likely to be expanded later. In that case, this will be renamed/refactored accordingly.